packages/tendermint-rpc/src/dates.ts

```diff
@@ -25,6 +26,29 @@ export function toRfc3339WithNanoseconds(dateTime: ReadonlyDateWithNanoseconds):
   return `${millisecondIso.slice(0, -1)}${nanoseconds.padStart(6, "0")}Z`;
 }
 
+export function fromSeconds(seconds: number, nanos = 0): DateWithNanoseconds {
```

Comment:
I see how this can be useful in repos such as ts-relayer. I did make a few errors before I got the timestamps calculated properly. :+1: